operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfigurations.java

```diff
@@ -0,0 +1,93 @@
+package org.bf2.operator.operands;
+
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.quarkus.runtime.Startup;
+import org.bf2.operator.resources.v1alpha1.ManagedKafka;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.function.UnaryOperator;
+
+/**
+ * Loads the properties for each instance profile from config files and from the Quarkus Config.
+ * <p>
+ * Default properties will have the prefix managedkafka.
+ * <p>
+ * Properties in each instance file will omit this prefix.
+ * <p>
+ * Properties overlayed from the Quarkus Config will be prefixed accordingly - managedkafka.kafka... standard.kafka...
+ */
+@Startup
+@ApplicationScoped
+public class KafkaInstanceConfigurations {
+
+    private static final String MANAGEDKAFKA = "managedkafka";
```

Comment:
why not remove this prefix everywhere, or since the property files are already named differently have the same prefix on all?

Reply:
> why not remove this prefix everywhere

The original intent of the managedkafka prefix was to organize our own properties in the application.properties file.  Stuff with that prefix would be different than properties that were for configuring internal things like polling intervals.

We also need to make sure that any changes to the property prefix, or even what I've done to introduce a different standard set of property doesn't conflict with anything that is already part of an SOP.

> or since the property files are already named differently have the same prefix on all

You could have the same, but I prefer having no prefix in those files than having managedkafka. - because if you were to actually override them it would need to be standard. or trial. etc.

Reply:
seems like instead of depending on `application.properties` if you introduce a `default.properties` then with prefix one place vs no prefix other will fix itself?

Reply:
You could along with the change to move all such properties on to the kafkainstanceconfiguration, but again it didn't seem necessary at this juncture.

Reply:
I tend to agree that having the configs for this out of `application.properties` is easier for reference purposes. But, I also like the option to override using the environment or system properties.